tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -185,37 +185,66 @@ namespace Datadog.Trace.AppSec
         public static void TrackUserLoginSuccessEvent(string userId, System.Collections.Generic.IDictionary<string, string> metadata) { }
```

Comment:
If you're publishing a new NuGet, I think you should have public API tests for the new NuGet too (documenting public API and referenced assemblies, same as for these)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3774/commits/fe3306300a9d7b3bccc4a5e4ddecd0b1f28a4c9a